specification/softwareplan/resource-manager/readme.go.md

```diff
@@ -17,3 +17,12 @@ Please also specify `--go-sdks-folder=<path to the root directory of your azure-
 ```yaml $(tag) == 'package-2019-06-01-preview' && $(go)
 output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2019-06-01-preview/$(namespace)
 ```
+
+### Tag: package-2019-12-01 and go
+
+These settings apply only when `--tag=package-2019-12-01 --go` is specified on the command line.
+Please also specify `--go-sdks-folder=<path to the root directory of your azure-sdk-for-go clone>`.
+
+```yaml $(tag) == 'package-2019-12-01' && $(go)
+output-folder: $(go-sdk-folder)/services/preview/$(namespace)/mgmt/2019-12-01/$(namespace)
```

Comment:
Please add a `preview` suffix to the api-version, ie, change the output folder to:
`$(go-sdk-folder)/services/preview/$(namespace)/mgmt/2019-12-01-preview/$(namespace)`

Reply:
Sorry, to confirm, is this version a stable version? If this is stable, the output-folder should be:
`$(go-sdk-folder)/services/$(namespace)/mgmt/2019-12-01/$(namespace)`
If this is preview, please ignore this comment

Reply:
This is a stable version. let me update the path